user/models.py

```diff
@@ -1,11 +1,14 @@
 import uuid
 from django.db import models
 from django.contrib.auth.models import AbstractUser
+from django.core.validators import FileExtensionValidator
 
 
 class User(AbstractUser):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     enable_notifications = models.BooleanField(default=True)
+    avatar = models.FileField(upload_to='user/', blank=False, null=True, validators=[
```

Comment:
what options we have for resizing the images on uploading? (dont implement yet)

Reply:
I have used FileField for saving image if we need to resize image before saving then we need to use imageField from Pillow instead of FileField. By using ImageField we just need to pass height and width as a param and it should work.

Reply:
@usmandap lets do that for avatar only